### ISSUE TITLE
[DISCUSS] Fork this package? Repo not maintained anymore

### ISSUE THREAD

**Issue by fungilation** [leecade/react-native-swiper#893]
Issues and PRs are largely unattended for months now, and the owner just updates README with [recruitment ads](https://github.com/leecade/react-native-swiper/commit/81dd7be33e437e301f4504e398e6d346b5a7de06). [My app](https://medium.com/wonderswipe/rethink-mobile-search-10-100x-faster-introducing-wonderswipe-6f2ff0d0e667)'s main navigation is built around this library so I have a vested interest in improving and maintaining this, with community contribution and merging useful PRs on queue here. [My own fork](https://github.com/fungilation/react-native-swiper) is working well up to current RN 0.57.0 on both iOS and Android.

If we move to a new repo under a new name, what name should it be? What PRs should be merged?

My idea for new name: react-native-snap-scrollview or react-native-snap-swiper

Discuss.

**Comment by kristerkari** [comment]
I think that if this is getting forked into something that is maintained, it would be a good idea to divide the issues to a list of bugs and a list of features, and then create a proposed roadmap for the features/fixes that would be done next.

I think that I could help out, at least for a while, to get that done.

**Comment by fungilation** [comment]
Fragmentation is not good for OSS, so working together is for the best. Whether in this repo or new. 

@arribbar, is new repo actually better? Do you have npm access for this package?

**Comment by pascaloliv** [comment]
I agree with @fungilation - fragmentation is bad. What is @leecade 's point of view on this situation ?

**Comment by arribbar** [comment]
I have access to npm package and I can add any maintainer. I just need your npm username.
If we keep this repo, we'll need to clean all issues and PRs and it will takes some time to do so.
But if we change it, all users will need to change for the new one...

It might be better to keep going with this project. We just need to have admin rights on github project.

**Comment by arribbar** [comment]
I can make a new release if you want to this week or the next one
Just tell me which PRs need to be merged in the new release and I'll do it

**Comment by pascaloliv** [comment]
@kristerkari : I said 2.0.0 because I anticipated some major changes yeah. But I'm not sure about how big the diff is between the `leecade/react-native-swiper` and @fungilation 's one.

+1 : Good idea to flag some important issues / merge requests / features for the next milestone | release.

**Comment by arribbar** [comment]
I agree with you @kristerkari 
We'll just need to do some cleaning in issues and PRs but that's ok

I have just created next release tag for now

I'll let you decide which PRs need to be merged. I'll do a check next week
If you have other ideas, feel free to share them :)

**Comment by fungilation** [comment]
Title changed. Let's nominate PRs that's already been tested to work, in your own app or test repo. For merging for next release.

I nominate #789 #684

@arribbar, thanks for continuing to help maintaining this package. You said "We just need to have admin rights on github project." Does that mean you don't have all github permissions and if so, what are you missing that'd warrant forking to a new repo?

**Comment by arribbar** [comment]
Well, I cannot add/remove any maintainers. It would be better if some users can have these rights.
I asked @leecade if I could have them. Let's see ...

**Comment by fungilation** [comment]
Ya let's see. If you get full admin access then no reason for us to rename the package.

**Comment by arribbar** [comment]
Release 1.5.14 just done with #789 #684 merged

**Comment by arribbar** [comment]
Another question for you guys,
I am a bit lost between index.d.ts, a typescript file, and index.js.flow, a flow file. Should we keep both files ? How can we update/test these files ?

Another one which would be great, I'd like to add a how to contribute file. Do you have some ideas about what to put in this file ? And some time to write it ? :)

And what do we do about all these PRs and issues ?

**Comment by fungilation** [comment]
On PRs, I was asking people to nominate PRs they actually use or want.

**Comment by sfratini** [comment]
I siwtched to this one: https://github.com/archriss/react-native-snap-carousel

Seems to have more traffic, features and less issues.
